internal/log/log.go

```diff
@@ -0,0 +1,40 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	"log"
+)
+
+// New creates a Logger instance.
+func New(w io.Writer, prefix string) *Logger {
```

Comment:
My 2: pick a logger you like and go with it, no need for the indirection.

Reply:
I usually use Zap, but I it seemed I just through this together and have been running with it :man_shrugging: :smile:

Reply:
I like Zap too :)

The main idea here is that we don't need the configurability for the challenge, stick a real logger wherever you want it and we're good to go.